bundles/AdminBundle/Resources/public/js/pimcore/document/edit.js

```diff
@@ -391,11 +391,17 @@ pimcore.document.edit = Class.create({
 
             for (var i = 0; i < editables.length; i++) {
                 try {
-                    if (editables[i].getName() && !editables[i].getInherited()) {
-                        editableName = editables[i].getName();
-                        values[editableName] = {};
-                        values[editableName].data = editables[i].getValue();
-                        values[editableName].type = editables[i].getType();
+                    if(only == "validate") {
```

Comment:
We can collect all editables wich are configured as required already on initialization here: 
https://github.com/pimcore/pimcore/blob/98f8f80933856fad19bba41678caa976e9562b86/bundles/AdminBundle/Resources/public/js/pimcore/document/edit/startup.js#L146

I'd say we can just reference them in another array, something like that should be sufficient: 
```js
try {
	let editable = getEditable(editableConfigurations[i]);
	editables.push(editable);
	if(editable.isRequired()) {
		requiredEditables.push(editable)
	}
} catch (e) {
```